### ISSUE TITLE
Documentation: ambiguous interchangeable use of "white noise" and "Wiener process"

### ISSUE THREAD

**Issue by aterenin** [JuliaDiffEq/DiffEqDocs.jl#211]
In the documentation it is not clear what is meant by white noise.

It's possible that this refers to a standard Brownian motion, which is an R^d-valued stochastic process which is characterized as a Levy process with continuous sample paths.

It's also possible that this refers to a (standard) white noise process, which can be defined as a Gaussian measure taking values in the Banach space of Schwarz distributions S'(\R^d). The covariance operator is given by the action of a Dirac delta (see Example 3.88 in M. Hairer's SPDE notes). One can use an isometric property to define white noise integrals (see Section 3.1 in M. Lifshits, Lectures on Gaussian Processes), from which one can conclude that the integral of this stochastic process is function-valued, and is indeed a standard Brownian motion.

These two kinds of random functions are not the same. Moreover, it's possible that one very well might want to define an ODE (in the weak sense) driven by white noise - this doesn't currently seem to be supported, but the documentation seems to suggest it might be.

**Comment by ChrisRackauckas** [comment]
Are you referring to this line? http://docs.juliadiffeq.org/latest/features/noise_process.html#Wiener-Process-(White-Noise)-1 Then yes that can be made more clear, but it probably doesn't need much because the package docs should be kept concise and the true mathematical description can generally be thought to be known (or cited). 

>define an ODE (in the weak sense) driven by white noise

You mean a random ordinary differential equation driven by white noise? Is it different than what's shown in that section?

**Comment by aterenin** [comment]
Yes, that line, though this is not the only ambiguous one.

My point is that the random ODE

dX(\omega, t) / dt = f(X(\omega, t), t) + B(\omega,t)

where B is a Brownian motion and the random ODE

dX(\omega, t) / dt = f(X(\omega, t), t) + W(\omega,t)

where (by abuse of notation) W is temporal white noise, with both equations interpreted in the weak sense, are not the same. In appropriate cases the latter is an Ito SDE, whereas the former is not. The documentation is written as if they are identical.

**Comment by ChrisRackauckas** [comment]
Interesting, what's a case where they are different? Do you have a good reference on cases where they are different? I thought that under standard interpretations of the integral Wiener processes had the same properties of a Brownian motion and so in all fields they were considered equivalent terms, with Wiener processes being the rigorous mathematical definition for the physical process.

I guess we should just make sure to be clear that it's a Wiener process, following the parlance of Kloeden's work.

**Comment by aterenin** [comment]
Sorry, I should be more clear - my previous comment used poor notation for temporal white noise. The issue here isn't about nitpicking details of definitions - that is a discussion worth having, albeit in a different venue. It's that white noise, however you formalize it, is the _derivative_ of Brownian motion, and not Brownian motion itself. 

Thus the integral of a temporal white noise process

B(T) =  \int_0^T dWN(t)
B(T) ~ N(0, T)

is a Brownian motion. On the other hand, the integral of a Brownian motion 

X(T) = \int_0^T B(t) dt
X(T) ~ N(0, T^3 / 3)

is not a Brownian motion but is some other stochastic process.

**Comment by ChrisRackauckas** [comment]
Yes, that is true. So where is it used incorrectly? The RODE default uses a white noise process whose integral is the Wiener process W(t). That seems to be how it's written? The only spot that seems to need some clarification still is http://docs.juliadiffeq.org/latest/features/noise_process.html#Wiener-Process-(White-Noise)-1 right? Instead of keeping the discussion vague, let's make it specific and actionable.

**Comment by aterenin** [comment]
Yes, that one. In particular

```
Wiener Process (White Noise)
The WienerProcess, also known as Gaussian white noise, Brownian motion, or the noise in the Langevin equation, is the stationary process with distribution N(0,t). The constructor is:
```

could be rewritten

```
Wiener Process
The Wiener Process, also known as the integral of Gaussian white noise, Brownian motion, or the noise in the Langevin equation, is the stationary process with distribution N(0,t). The constructor is:
```

On the same page, the term is also used throughout `Direct Construction Example`, and it's not immediately clear to me in which sense the term meant - I suspect the term white noise should simply be replaced therein with Brownian motion - but I should really sit down and look at it later to figure it out.

**Comment by ChrisRackauckas** [comment]
That sounds like a good change to me.
